sdk/serialize-utils/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "solana-serialize-utils"
+description = "Solana helpers for reading and writing bytes."
+documentation = "https://docs.rs/solana-serialize-utils"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+solana-instruction = { workspace = true, default-features = false, features = [
+    "std",
+] }
+solana-pubkey = { workspace = true, default-features = false }
+solana-sanitize = { workspace = true }
+
+[dev-dependencies]
+bincode = { workspace = true }
+borsh0-10 = { workspace = true }
```

Comment:
Can you change this to use borsh v1 instead? You'll just need to update the tests to use `borsh` instead of `borsh0_10`, and that way we avoid a dependency on an old crate